assume/units/dst_components.py

```diff
@@ -0,0 +1,819 @@
+# SPDX-FileCopyrightText: ASSUME Developers
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import pyomo.environ as pyo
```

Comment:
in this file please check again if everyhting is sufficiently documented and clear enough so users and future developers can follow up with it

Reply:
Checked and updated!